audformat/core/utils.py

```diff
@@ -474,32 +483,38 @@ def expand_file_path(
         index: pd.Index,
         root: str,
 ) -> pd.Index:
-    r"""Expand relative path in index with root.
+    r"""Expand path in index with root.
+
+    It applies :func:`os.path.normpath`
+    to the provided path ``root``,
+    adds a file separator at its end
+    and puts it in front of the file path in the index.
 
     Args:
-        index: index with relative file path conform to
+        index: index with absolute or relative file path
```

Comment:
Feels a bit strange to even highlight that the file path can be absolute. I mean yes, it can be, but expanding in that case doesn't really make sense. So I think I would either stick to what we had before or just say `index conform to ...`

Reply:
I switched to `index conform to ...`